_config.yml

```diff
@@ -1,6 +1,27 @@
 title: Site title
 
+# GitHub information
```

Comment:
I like all the explanatory comments in here and in the other config files

Reply:
agree